.github/workflows/test-csharp.yml

```diff
@@ -19,8 +19,8 @@ jobs:
 
       - name: Build
         run: dotnet build
-        working-directory: ./bindings/csharp
+        working-directory: ./bindings/csharp/net8.0
```

Comment:
We could test both net8.0 and net40

Reply:
I can try, but not sure if the tool chain can support both of them at the same time